docs/user-guide/concepts.md

```diff
@@ -606,3 +606,14 @@ namespace and turn it back into a subnamespace.
 This is the [tree label](#basic-labels) which is documented above. It is set by
 HNC on namespaces and (unless `managed-by` was set first) cannot be modified by
 anyone other than HNC.
+
+<a name="included-namespace-label">
+
+#### hnc.x-k8s.io/included-namespace (label on namespaces)
+
+***This label is only available in HNC v0.9 and later***
+
+HNC's mutating webhook sets this label on non-excluded namespaces and removes
+this label from excluded namespaces by default. This allows HNC's object
+validating webhook to fail close on all namespaces with this label. See [Excluding namespaces from HNC](how-to.md#admin-excluded-namespaces)
+for more information.
```

Comment:
Most users won't know what "fail closed" means. I'd leave that detail out and would say something about:
* Some webhooks will only operate in namespaces with this label
* This protects your cluster's critical namespaces from problems with HNC